Anagrams/src/ua/goit/alg/TestAnagrams.java

```diff
@@ -0,0 +1,36 @@
+package ua.goit.alg;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.Test;
+
+public class TestAnagrams {
+
+	@Test
+	public void twoWords() {
+		//Copy this to console:hello world!
+		String expected = " olleh !dlrow";
+		assertOutput(expected);
+	}
+
+	@Test
+	public void threeWords() {
+		//Copy this to console:My name is:
+		String expected = " yM  eman :si";
+		assertOutput(expected);
+	}
+
+	private void assertOutput(String expected) {
+		ByteArrayOutputStream baos = new ByteArrayOutputStream();
+		PrintStream ps = new PrintStream(baos);
+		PrintStream old = System.out;
+		System.setOut(ps);
+		Anagrams.Anagram(System.in);
```

Comment:
Does that mean you're entering words from the keyboard? That's a no-no in unit tests. Please use StringReader here